index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Isaac Jung's Resume</title>
+    <link rel='stylesheet' href='styles.css' />
+</head>
+
+<body>
+    <div class="page u-clearfix">
+        <main>
+            <div class="content01">
+                <h1>Graphic Designer & Sound Artist</h1>
+                <p><u>Isaac Jung</u> is based in Seoul, Republic of Korea.
+                    A flexible creator skilled in conceptualizing visual systems, analyzing product UIUX,
+                    sound editing, and site specific sonic installations He is also a traveler,
+                    an avid barista, a part-time composer, a devoted table tennis player,
+                    and an eager learner in all the arts.</p>
+            </div>
+            <div class="content02">
+                <h2>Experience</h2>
+                <h3>K: The Portfolio Lab­­—Rebranding Freelance</h3>
+                <p class="dates">SEP, 2018–Oct, 2018</p>
```

Comment:
Great use of a class name here to collectively style all dates in a similar format. I also note that you're using [using all caps for presentational purposes](https://mica-web.github.io/learn/html/best-practices#do-not-use-all-caps-in-your-markup-content-for-presentational-purposes), which has lead to SEP being in all caps, but Oct not. Better to do this in CSS where it can be consistent in all cases. If you added:
`text-transform: uppercase;`
to your `dates` class, it would handle this for you 👍🏻